src/Perpetuum/Services/Channels/ChannelMessageHandler.cs

```diff
@@ -10,25 +10,38 @@ namespace Perpetuum.Services.Channels
 {
     public static class ChannelMessageHandler
     {
-        private const string WELCOME_CHANNEL_NAME = "General chat";
+        private const string WELCOME_CHANNEL_NAME = "Recruitment";
+        private const string HELP_CHANNEL_NAME = "regchannel_help";
         private const string SENDER_CHARACTER_NICKNAME = "New Player Announcer";
 
+        public static void SendNewPlayerTutorialMessage(IChannelManager channelManager, string newCharacterName)
+        {
+            var test = channelManager.Channels.All(channel => channel.Name.ToLower().Contains("help"));
+
+            SendMessage(channelManager, HELP_CHANNEL_NAME, "NewPlayerEnteredTheGame", newCharacterName);
+        }
+
         public static void SendWelcomeMessageExitTutorial(IChannelManager channelManager, string newCharacterName)
+        {
+            SendMessage(channelManager, WELCOME_CHANNEL_NAME, "NewPlayerFinishedTheTutorial", newCharacterName);
```

Comment:
I have a feeling that these two strings are the wrong way around. Surely you should send the FinishedTutorial message when the tutorial is finished, and the EnteredGame when they exit the island?

I realise they're both sent at the same time and that it's merely semantics, but to read this by itself is confusing, to me at least.